src/wp-admin/includes/class-theme-upgrader.php

```diff
@@ -123,56 +123,69 @@ public function check_parent_theme_filter( $install_result, $hook_extra, $child_
 		// Check to see if we need to install a parent theme
 		$theme_info = $this->theme_info();
 
-		if ( ! $theme_info->parent() )
+		if ( ! $theme_info->parent() ) {
 			return $install_result;
+		}
 
 		$this->skin->feedback( 'parent_theme_search' );
 
 		if ( ! $theme_info->parent()->errors() ) {
-			$this->skin->feedback( 'parent_theme_currently_installed', $theme_info->parent()->display('Name'), $theme_info->parent()->display('Version') );
+			$this->skin->feedback( 'parent_theme_currently_installed', $theme_info->parent()->display( 'Name' ), $theme_info->parent()->display( 'Version' ) );
 			// We already have the theme, fall through.
 			return $install_result;
 		}
 
 		// We don't have the parent theme, let's install it.
-		$api = themes_api('theme_information', array('slug' => $theme_info->get('Template'), 'fields' => array('sections' => false, 'tags' => false) ) ); //Save on a bit of bandwidth.
+		$api = themes_api(
+			'theme_information',
+			array(
+				'slug'   => $theme_info->get( 'Template' ),
+				'fields' => array(
+					'sections' => false,
+					'tags'     => false,
+				),
+			)
+		); //Save on a bit of bandwidth.
```

Comment:
This comment should be moved up.

Reply:
Seems odd but in the same place upstream:
https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-admin/includes/class-theme-upgrader.php#L150